### PR TITLE
[CDAP-13696] add tooltip for operation name and modify column name to be singular

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-13696
Build: https://builds.cask.co/browse/CDAP-URUT17

### CODE REVIEW

**Comment by tonybach** [comment]
Should we add tooltips for other fields as well, e.g. description?

**Comment by elfenheart** [comment]
I added it to description. The others cannot be truncated

File: cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/OperationsTable.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default class OperationsTable extends Component {
           <div></div>
           <div>Input</div>
           <div>Input fields</div>
```
Comment (bdmogal) [inline cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/OperationsTable.js]: should we do i18n for these?

**Review by ajainarayanan — COMMENTED** [review]
Code LGTM 👍

**Review by ajainarayanan — COMMENTED** [review]
👍 code lgtm

**Comment by elfenheart** [comment]
demoed to @bdmogal

**Pull request merged**